sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredentialBuilder.java

```diff
@@ -10,7 +10,12 @@
  * This mechanism for Visual Studio authentication has been replaced by the {@link VisualStudioCodeCredential}/>.</p>
  *
  * @see SharedTokenCacheCredential
+ * @deprecated This credential was originally created to support authentication with Visual Studio. Since Visual Studio
```

Comment:
Visual Studio Code I believe.

Reply:
the original text is correct: this credential enabled interop with VS, not VS Code